templates/summary_manager/handle_skills.html.twig

```diff
@@ -0,0 +1,79 @@
+{% extends 'summary_manager/layout.twig' %}
+{% block summary_content %}
+    <section class="l__wrapper--medium">
+
+        <h2>Comptétences</h2>
+
+        <h3>Ajouter une compétence</h3>
+
+        <div class="l__row l__row--h-stretch l__row--top l__tablet--col b__nudge--bottom-large">
+            {{ form_start(form) }}
+
+            {{ form_widget(form.skill_search) }}
+            {% for skill in form.skills %}
+                <div class="summary_skill form__row">
+                    {{ form_widget(skill) }}
+                </div>
+            {% endfor %}
+
+            <div id="summary_skills_list">
+                {% for skill in form.vars.data.skills %}
+                    <div><span class="summary-skill-remove">X</span> {{ skill.name }}</div>
+                {% endfor %}
+            </div>
+
+            <div>
+                <button type="submit" id="edit-skills" class="btn btn--blue">
+                    Enregistrer
+                </button>
+                <a href="{{ path('app_summary_manager_index') }}" class="btn">
+                    Annuler
+                </a>
+            </div>
+            {{ form_end(form) }}
+        </div>
+
+    </section>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jqueryui/ui/jquery-ui.js') }}></script>
+    <script type="text/javascript">
+        $(document).ready(function() {
+            // Prevent submitting form by pressing enter
+            $(window).keydown(function(event){
+                if(event.keyCode == 13) {
+                    event.preventDefault();
+                    return false;
+                }
+            });
+
+            // Add skill autocomplete
+            $("#summary_skill_search").autocomplete({
+                source: "{{ path('app_summary_manager_skill_autocomplete') }}",
+                minLength: 2,
+                messages: {
+                    noResults: '',
+                    results: function() {},
+                }
+            });
+
+            // Add skill by pressing enter
+            $("#summary_skill_search").keyup(function(e){
+                if(e.which == 13 || e.keyCode == 13){
+                    e.preventDefault();
+                    $('#summary_skills_list').append('<div><span class="summary-skill-remove">X</span> ' + $(this).val() + '</div>');
```

Comment:
You should add a prototype entry form so it can be submitted and added to the collection.